Failing test: DatabaseImportControllerTest.testDoPostParsePreview

After merging the security patches in master, we are getting the following error [in the CI](https://github.com/OpenRefine/OpenRefine/actions/runs/11494591933/job/31992369675) when running backend tests for the database extension:
```
 Error:  Tests run: 84, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 5.562 s <<< FAILURE! -- in TestSuite
Error:  com.google.refine.extension.database.DatabaseImportControllerTest.testDoPostParsePreview -- Time elapsed: 0.234 s <<< FAILURE!
Error: ng.AssertionError: expected [ok] but found [error]
```

This test passes locally on my side, so I am not sure what could be the cause.
I am disabling it for now so that it does not fail other builds (bf47ddf070e).

--- a/extensions/database/tests/src/com/google/refine/extension/database/DatabaseImportControllerTest.java
+++ b/extensions/database/tests/src/com/google/refine/extension/database/DatabaseImportControllerTest.java
@@ -195,7 +195,7 @@ public class DatabaseImportControllerTest extends DBExtensionTests {
         ObjectNode json = ParsingUtilities.mapper.readValue(result, ObjectNode.class);
 
         String status = json.get("status").asText();
-        System.out.println("response: " + json);
+        // System.out.println("json::" + json);
         Assert.assertEquals(status, "ok");
     }
 

--- a/extensions/database/tests/src/com/google/refine/extension/database/DatabaseImportControllerTest.java
+++ b/extensions/database/tests/src/com/google/refine/extension/database/DatabaseImportControllerTest.java
@@ -161,8 +161,7 @@ public class DatabaseImportControllerTest extends DBExtensionTests {
         Assert.assertEquals(status, "ok");
     }
 
-    // 2024-10-24 disabled for now as it fails in the CI (but not locally)
-    @Test(enabled = false)
+    @Test
     public void testDoPostParsePreview() throws IOException, ServletException {
 
         StringWriter sw = new StringWriter();

--- a/extensions/database/tests/src/com/google/refine/extension/database/DatabaseImportControllerTest.java
+++ b/extensions/database/tests/src/com/google/refine/extension/database/DatabaseImportControllerTest.java
@@ -195,7 +195,7 @@ public class DatabaseImportControllerTest extends DBExtensionTests {
         ObjectNode json = ParsingUtilities.mapper.readValue(result, ObjectNode.class);
 
         String status = json.get("status").asText();
-        // System.out.println("json::" + json);
+        System.out.println("response: " + json);
         Assert.assertEquals(status, "ok");
     }
 

